src/reporter.ts

```diff
@@ -0,0 +1,126 @@
+import { summary } from "@actions/core";
+
+import { PullRequestMetrics } from "./report/types";
+import { calculateAverage } from "./util";
+
+export const generateSummary = (
+  repo: { owner: string; repo: string },
+  metrics: PullRequestMetrics,
+): typeof summary => {
+  const prChart = `\`\`\`mermaid
+  pie title Pull Requests for the repository
+  "Open" : ${metrics.open}
+  "Merged" : ${metrics.merged}
+  "Closed" : ${metrics.closed}
```

Comment:
This is the current number of open PRs, and the total numer of merged/closed PRs of all time, correct?

Reply:
That's right